sim/core/aura.go

```diff
@@ -388,6 +392,11 @@ func (at *auraTracker) AddAura(sim *Simulation, newAura Aura) {
 func (at *auraTracker) RemoveAura(sim *Simulation, id AuraID) {
 	aura := &at.auras[id]
 
+	if aura.ID == 0 {
+		// rarely happens if advance() triggers actions that result in RemoveAura() calls
```

Comment:
This happens during the rogue tests, at some point: Adrenaline Rush fades, changes energy, calls doRotation() and the resulting chain ends in a RemoveAura() call, followed by continuing advance(), which happens to remove that aura again.